cmd/faFormat/faFormat.go

```diff
@@ -66,8 +66,8 @@ func faFormat(s Settings) {
 			fasta.ToUpper(records[i])
 		}
 		if s.RevComp {
-			fasta.ReverseComplement(records[i])
```

Comment:
This might've been switched by mistake. It's been a while since I looked at this but I vaguely remember faFormat failing a tests somewhere in the fasta package and the switch was an artifact of a copy/paste debugging. Can switch it back since to your point the doesnt seem to matter